cmd/cli.go

```diff
@@ -1,42 +1,62 @@
-package shelLF
+package cmd
 
 import (
 	"flag"
-	"log"
 	"os"
 	"time"
+
+	"github.com/LetsFocus/goLF/logger"
 )
 
-func NewCLI(toolName string) *CLI {
+func NewCLI() *CLI {
 	commandMap := make(map[string]*Command)
-	return &CLI{toolName: toolName, commands: commandMap}
+	return &CLI{toolName: "myTool", commands: commandMap}
+}
+
+func (cli *CLI) SetCLIName(toolName string) {
+	cli.toolName = toolName
+}
+
+func (cli *CLI) SetCLIVersion(toolVersion string) {
+	cli.version = toolVersion
 }
 
 func (cli *CLI) AddCommand(cmd Command) {
 	cli.commands[cmd.Name] = &cmd
 }
 
 func (cli *CLI) printUsage() {
-	log.Printf("Usage: %s <command> [options]", cli.toolName)
-	log.Println("Available commands:")
+	logger := logger.NewCustomLogger()
+	logger.Infof("Usage: %s <command> [options]", cli.toolName)
+	logger.Infof("Available commands:")
```

Comment:
remove new logger and also use fmt.Println or Printf here  because logs will give time as well which is not needed